[DOCS] npm cli help does not fit the real npm behavior

### Is there an existing issue for this?

- [X] I have searched the existing issues

### This is a CLI Docs Problem, not another kind of Docs Problem.

- [X] This is a CLI Docs Problem.

### Description of Problem

```
# npm search --help
Search for packages

Usage:
npm search [search terms ...]

Options:
[-l|--long] [--json] [--color|--no-color|--color always] [-p|--parseable]
[--no-description] [--searchopts <searchopts>] [--searchexclude <searchexclude>]
[--registry <registry>] [--prefer-online] [--prefer-offline] [--offline]

aliases: find, s, se

Run "npm help search" for more info
```
As you can see all parameters are in square brackets, so they are optional.
Doing search:
```
# npm search --registry=https://my.local.registry
npm ERR! search must be called with arguments
```
??? With what arguments ? Help does not show any arguments requirement.
Trying another help (as stated by `npm` itself - `Run "npm help search" for more info`):
```
# npm help search
npm ERR! help process exited with code: ENOENT
```
WTF ??? Where is normal help ?

Versions:
npm-9.8.1
nodejs-18.18.2

### Potential Solution

_No response_

### Affected URL

_No response_

--- a/lib/commands/search.js
+++ b/lib/commands/search.js
@@ -8,7 +8,6 @@ class Search extends BaseCommand {
   static description = 'Search for packages'
   static name = 'search'
   static params = [
-    'long',
     'json',
     'color',
     'parseable',
@@ -22,7 +21,7 @@ class Search extends BaseCommand {
     'offline',
   ]
 
-  static usage = ['[search terms ...]']
+  static usage = ['<search term> [<search term> ...]']
 
   async exec (args) {
     const opts = {

--- a/tap-snapshots/test/lib/docs.js.test.cjs
+++ b/tap-snapshots/test/lib/docs.js.test.cjs
@@ -4065,11 +4065,11 @@ exports[`test/lib/docs.js TAP usage search > must match snapshot 1`] = `
 Search for packages
 
 Usage:
-npm search [search terms ...]
+npm search <search term> [<search term> ...]
 
 Options:
-[-l|--long] [--json] [--color|--no-color|--color always] [-p|--parseable]
-[--no-description] [--searchlimit <number>] [--searchopts <searchopts>]
+[--json] [--color|--no-color|--color always] [-p|--parseable] [--no-description]
+[--searchlimit <number>] [--searchopts <searchopts>]
 [--searchexclude <searchexclude>] [--registry <registry>] [--prefer-online]
 [--prefer-offline] [--offline]
 
@@ -4078,14 +4078,13 @@ aliases: find, s, se
 Run "npm help search" for more info
 
 \`\`\`bash
-npm search [search terms ...]
+npm search <search term> [<search term> ...]
 
 aliases: find, s, se
 \`\`\`
 
 Note: This command is unaware of workspaces.
 
-#### \`long\`
 #### \`json\`
 #### \`color\`
 #### \`parseable\`

